### PR TITLE
Add symfony 4 compatibility

### PR DESCRIPTION
**Pull request by JordanProtin**
Hey !

I updated the composer.json file in order to add the symfony 4 compatibility.
Ty.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,9 +13,9 @@
     ],
     "require": {
         "php": ">=5.3.2",
-        "symfony/framework-bundle": "2.*|~3.0",
-        "symfony/yaml": "2.*|~3.0",
-        "google/apiclient": "1.*|2.*"
+        "symfony/framework-bundle": "2.*|~3.0|^4.0",
+        "symfony/yaml": "2.*|~3.0|^4.0",
+        "google/apiclient": "1.*|2.*|^2.0"
```
Comment (Nyholm) [inline composer.json]: 2.* means `>=2.0.0 && <3.0.0`
^2.0 means `>=2.0.0 && <3.0.0`
Which is exactly the same. Please revert this change.

Comment (JordanProtin) [reply composer.json]: Indeed, I totally agree with you, it's a careless mistake.. I reverted this !

**Comment by JordanProtin** [comment]
Sure,

I could test this by adding manually this bundle and by putting the good versions.
Please note that actually, sf4 require "symfony/framework-bundle": "^4.0" and "symfony/yaml": "^4.0" 

Thank You.
